packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall.controller.js

```diff
@@ -13,21 +13,46 @@ export default /* @ngInject */ (
   $stateParams,
   $http,
   atInternet,
-) => {
+) {
+  const self = this;
+
   $scope.selectedBlock = null;
```

Comment:
As you are now using self, maybe move old $scope variables inside self as well

Reply:
I'll did into the next feature (which is coming very soon)